pkg/apis/storage/validation/validation.go

```diff
@@ -38,6 +39,17 @@ const (
 	maxVolumeErrorMessageSize     = 1024
 )
 
+// PodInfoOnMountVersions indicates the CSI volume driver
+// requires additional pod information (like podName, podUID, etc.)
+// during mount operations. "v1" is the only supported version now.
+var supportedPodInfoOnMountVersions = sets.NewString(string("v1"))
+
+// VolumePluginMechanism announces what mechanism underlies volume plugins.
+// It is set by Kubelet. It is used by the attach/detach controller, which
+// needs to know how to perform attachments. The allowed values are "in-tree"
+// and "csi".
+var supportedVolumePluginMechanisms = sets.NewString(string("in-tree"), string("csi"))
```

Comment:
Please use the constants for these strings.

Reply:
Sure